src/tools/rustc-std-workspace-std/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "rustc-std-workspace-std"
+version = "1.0.0"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
+license = 'MIT OR Apache-2.0'
+description = """
+Hack for the compiler's own build system
+"""
+edition = "2018"
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+std = { path = "../../libstd" }
```

Comment:
@alexcrichton could you add this dir to the rust-src dist package as well? We'll likely need that to let xargo build libtest.

Reply:
Good catch! Added this in.